### DISCUSSION TITLE
How can we find out a key translated by machine or human ?

### DISCUSSION THREAD

**Discussion by ariaieboy** [Laravel-Lang/common#85] [General]
We use machine translations for the new keys. That's cool but do we have a mechanism to find out which key is translated by machine and which is translated by humans so we as referents can double check the machine translations?
For example, I re-checked all Persian translations a while ago and made some adjustments so we have a precise translation across the entire translation files and keys.
How can I check the machine translated keys unless checking each PR made by the machine?

**Comment by andrey-helldar** [comment]
The keys themselves are not tracked in any way, otherwise the scope of the project would be increased.

There are two ways to track machine translation:

1. When new keys are added to the project, whether by downloading files or by manually adding, I always use the mention (@Laravel-Lang/laravel-lang) of the team so that you receive an email. If it doesn't, check your notification settings.
2. You can find PR yourself by searching for the "source" keyword: https://github.com/Laravel-Lang/lang/pulls?q=is%3Apr+is%3Aclosed+source

Thus, you will receive the same PR that adds new keys. For example, [this one](https://github.com/Laravel-Lang/lang/pull/3171/files).

**Comment by Oleksandr-Moik** [comment]
I can provide one more option for searching in editor. For example I use PhpStorm, but it can be applied to another editor as well.

we use the word label, which is used when changing a translation in a comment or pull request. For example, for Ukrainian the word label it is `[uk]`. 

So I type it into the git search and see which commits changed the translations for that language.

![image](https://user-images.githubusercontent.com/50796878/232411085-8aec14a7-7615-4540-9be4-af1d7418e2f7.png)
